rox_bringup/launch/bringup_sim_launch.py

```diff
@@ -282,6 +284,11 @@ def generate_launch_description():
             description='Set this argument to True if you have an UR arm with DC variant'
         )
 
+    declare_franka_pwr_variant_cmd = DeclareLaunchArgument(
+            'use_dc_control_box', default_value='False',
+            description='Set this argument to True if you have an UR arm with DC variant'
```

Comment:
Change:

`if you have an UR arm with DC variant`

to 

`if you have an arm with DC variant or smaller controller box`